app/models/agents/twitter_user_agent.rb

```diff
@@ -82,6 +88,10 @@ def starting_at
       end
     end
 
+    def choose_home_time_line?
+      interpolated[:choose_home_time_line] != "false"
```

Comment:
We need to think about existing `TwitterUserAgent`s here. If I am not mistaken this would cause existing agents (that do not have the `choose_home_time_line` set) to return the home time line and not return the tweets of a user as they did previously.

I see too options, either inverse the condition to `interpolated[:choose_home_time_line] == "true"` or add a migration that sets `choose_home_time_line` to `false` for all existing `TwitterUserAgent`s